### ISSUE TITLE
Add validation rules to component's context object

### ISSUE THREAD

**Issue by codekiln** [wearebraid/vue-formulate#149]
I'm trying to implement a custom label Slot Component which will show an aria-hidden asterisk next to the label only if it is required. Currently, [I've implemented it like this](https://github.com/codekiln/vue-playground/blob/master/vue-formulate/formulate-custom-slot1/src/components/MyCustomLabel.vue#L31), 

```vue
<template>
  <label :for="context.id">
    <span v-if="isRequired" class="field-required" aria-hidden="true">*</span>
    {{ context.label }}
  ...
  </label>
</template>

<script>
export default {
  name: 'MyCustomLabel',
  //...
  computed: {
    isRequired () {
      return this.context.validationErrors.filter(msg => msg.includes('is required')).length
    }
  }
}
</script>
```

This works for now, but seems a little hacky. It would be nice if the list of validations supplied to the parent `<FormulateInput` were available on the `context` object. Is there already a better way of accomplishing this?

**Comment by justin-schroeder** [comment]
You're right. We should pass the parsed validation rules through on the context object...

Appreciate the issue. I'll tag this for future release.

**Comment by justin-schroeder** [comment]
This is now implemented in the `release/2.5.0` branch as a new `rules` property on the context object. The property contains an array of objects in the shape of: `{ ruleName: string, args: [] }` 👍
